src/applications/disability-benefits/all-claims/tests/utils.unit.spec.jsx

```diff
@@ -165,6 +165,7 @@ describe('526 helpers', () => {
     beforeEach(() => {
       // Replace fetch with a spy
       global.fetch = sinon.stub();
+      global.fetch.catch = sinon.stub();
```

Comment:
I think as a practice we should inject `apiRequest` into our functions instead of calling it directly from an import e.g. `function aFetchingFunction(someArgument, doFetch = apiRequest) { ... }`. Makes this sort of thing easier to test by removing the need to resolve promises in the unit test.

Reply:
I love this idea.